build/bin/build-server-rpm

```diff
@@ -46,6 +46,12 @@ prepare_specs() {
         local full_commit=$(git -C "${git_dir}" rev-parse HEAD)
         local short_commit=${full_commit:0:7}
 
+        if [[ ${spec_name} -eq "pmm-server" ]]; then
+            local pmm_git_dir=$(dirname $(find "${root_dir}/sources/pmm" -name .git | head -1))
```

Comment:
Do we really have to use `find`/`head`/`dirname` instead of `git -C ${root_dir}/sources/pmm/src/github.com/percona/pmm`?

And why not use `git_dir` and `full_commit`/`short_commit` and used below? If they are correct, and we can use them, we should. If they are not (for some reason I don't see), then we should not use them below this `if` block.

Reply:
@EvgeniyPatlan It would be great if you do not allow things like that slide during your reviews. We need to clean-up our builds scripts, not add more hacks and ad-hoc code.

Reply:
> Do we really have to use `find`/`head`/`dirname` instead of `git -C ${root_dir}/sources/pmm/src/github.com/percona/pmm`?

Such approach was implemented by @delgod. IMHO it's done for a simplifying code reading. 

> And why not use `git_dir` and `full_commit`/`short_commit` defined above and used below? If they are correct, and we can use them, we should. If they are not (for some reason I don't see), then we should not use them below this `if` block.

As I mentioned earlier - No new pmm.rpm is created. So variables have parameters for a building package. It's pmm-server in our case.

Reply:
>Such approach was implemented by @delgod. IMHO it's done for a simplifying code reading.

That approach was implemented for a general case when we need to find repository location for any component. This whole `if` block is the special case for the specific component and location we know already.

>As I mentioned earlier - No new pmm.rpm is created. So variables have parameters for a building package. It's pmm-server in our case.

Please re-read my comment. It has nothing to do with package building.

Reply:
Fixed - the command find has been omitted.